File: informatykaEuropejczyka/Rozdzial2/prog2_14.cpp
//algorytm sprawdzajacy, czy w ciagu znajduje sie liczba podzielna przez 7
#include <iostream>
using namespace std;
const int MAX = 100;

bool szukaj (int T[], int n)
{
 for (int i=0;i<n;i++)
  if (T[i]%7==0) return true;
 return false;
}

int main()
{
 int T[MAX], n;
 cout<<"podaj liczbe elementow tablicy: ";
 cin>>n;
 cout<<"podaj elementy tablicy:"<<endl;
 for (int i=0;i<n;i++)
 {
  cout<<"T["<<i<<"] = ";
  cin>>T[i];
 }
 if (szukaj(T,n)) cout<<"TAK"<<endl;
 else cout<<"NIE"<<endl;
 return 0;
}

File: informatykaEuropejczyka/Rozdzial2/prog2_14.py
#algorytm sprawdzajacy, czy w ciagu znajduje sie liczba podzielna przez 7

def szukaj (T):
    n=len(T)
    for i in range(n):
        if T[i]%7==0:
            return True
    return False

def szukaj2 (T):
    for k in T:
        if k%7==0:
            return True
    return False

#print(szukaj([7,5,6,4,5,3,4,8,2,3]))
#print(szukaj2([7,5,6,4,5,3,4,8,2,3]))
